spec/yaml/swiftnav/sbp/imu.yaml

```diff
@@ -115,3 +115,53 @@ definitions:
                       - 2: +/- 8g
                       - 3: +/- 16g
                       - 4: +/- 6g
+
+ - MSG_IMU_COMP:
+    id: 0x0905
+    short_desc: Compensated IMU data
+    desc: >
+      Data from the Inertial Measurement Unit, containing accelerometer and
+      gyroscope readings compensated for estimated errors and constant 
+      physical effects. The output is valid for the center of navigation.
+    fields:
+        - time:
+            type: u64
+            units: microseconds
+            desc: Microseconds since reference epoch
+        - flags:
+            type: u16
+            desc: Contains the applied compensation parameters and time synchronization mode
+            fields:
+              - 2-15: 
+                  desc: Reserved  
```

Comment:
I'd add flags for the following:

- gravity compensated
- earth rate compensated
- coriolis effect compensated
- accelerometer errors compensated
- gyroscope errors compensated
- maybe: transformed to VRP